<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Инспектор аэропортов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QCalendarWidget" name="cw_date">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="selectedDate">
       <date>
        <year>2017</year>
        <month>9</month>
        <day>14</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2016</year>
        <month>8</month>
        <day>15</day>
       </date>
      </property>
      <property name="maximumDate">
       <date>
        <year>2017</year>
        <month>9</month>
        <day>14</day>
       </date>
      </property>
      <property name="gridVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QRadioButton" name="rb_departure">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Вылет</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="11">
     <widget class="QTableView" name="tw_flightList"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lb_flightListLabel">
      <property name="text">
       <string>Список рейсов по выбранным аэропорту и дате</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cb_Airport">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="pb_showStat">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Отобразить загруженность аэропорта</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lb_aiportListLabel">
      <property name="text">
       <string>Список аэропортов</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1">
     <widget class="QRadioButton" name="rb_arrival">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Прилет</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="lb_dateLabel">
      <property name="text">
       <string>Дата рейса</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="pb_getFlightList">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Сформировать список рейсов</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cb_Airport</tabstop>
  <tabstop>cw_date</tabstop>
  <tabstop>rb_departure</tabstop>
  <tabstop>rb_arrival</tabstop>
  <tabstop>tw_flightList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
